### PR TITLE
Adding StringToAddress library.

### PR DESCRIPTION
**Pull request by Foivos**
Updates the dependancies.

### CODE REVIEW

File: src/util/StringToAddress.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            b = uint8(tmp[i]);
+            if ((b >= 97)&&(b <= 102)) b -= 87;
+            else if ((b >= 65)&&(b <= 70)) b -= 55;
+            else if ((b >= 48)&&(b <= 57)) b -= 48;
+            else return address(0);
+            iaddr |= uint160(uint256(b)<<(41-i<<2));
+        }
+        return address(iaddr);
+    }
+}
```
Comment (sammy1991106) [inline src/util/StringToAddress.sol:21]:
```suggestion
}

```

File: src/util/StringToAddress.sol [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+library StringToAddress {
+    function toAddress(string memory _a) internal pure returns (address){
```
Comment (sammy1991106) [inline src/util/StringToAddress.sol:6]: Please run prettier for format the code.

File: src/util/StringToAddress.sol [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+library StringToAddress {
```
Comment (sammy1991106) [inline src/util/StringToAddress.sol:5]: Did you take this from somewhere? We should have test cases for it.

**Review by sammy1991106 — CHANGES_REQUESTED** [review]

File: src/util/StringToAddress.sol (hunk 3 above)
Comment (Foivos) [reply src/util/StringToAddress.sol:5]: I took it from somewhere and enhanced it to support uppercase as well. Will add now.

**Review by sammy1991106 — APPROVED** [review]

**Review by re1ro — APPROVED** [review]

**Pull request merged**